hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -536,14 +535,6 @@ private void setOMNodeDetails(String serviceId, String nodeId,
     configuration.set(OZONE_OM_ADDRESS_KEY,
         NetUtils.getHostPortString(rpcAddress));
 
-    // Find the Ratis storage dir
-    String omRatisDirectory = OmUtils.getOMRatisDirectory(configuration);
-    if (omRatisDirectory == null || omRatisDirectory.isEmpty()) {
-      throw new IllegalArgumentException(HddsConfigKeys.OZONE_METADATA_DIRS +
-          " must be defined.");
-    }
-
-    omRatisStorageDir = OmUtils.createOMDir(omRatisDirectory);
```

Comment:
Why this code is completely removed, now ratis Storage creation/ check is done in a different place?

Reply:
Thank you @bharatviswa504 for catching this. This was by mistake. Added it back.